python/ray/experimental/serve/api.py

```diff
@@ -23,6 +24,12 @@ def _get_global_state():
     return global_state
 
 
+def _get_router_policy(queueing_policy_name):
```

Comment:
no need for this if we just ask user to pass in enum instance instead of string

Reply:
Please see the above comment